src/Invoice_GenUI/Invoice_GenUI/Models/Services/InvoiceService.cs

```diff
@@ -1,46 +1,33 @@
-﻿using System;
-using System.Net.Http;
-using System.Net.Http.Json;
+﻿using System.Net.Http;
 using System.Text;
 using System.Text.Json;
 using System.Threading.Tasks;
 
 namespace Invoice_GenUI.Models.Services
 {
-    public partial class InvoiceService : IInvoiceService
+    public partial class InvoiceService : BaseService, IInvoiceService
     {
         public async Task<bool> PutInvoice(InvoiceModel newInvoice)
         {
             bool result = false;
-            using (var client = new HttpClient())
+            using (var client = CreateHttpClient())
```

Comment:
:question: if there is a base service which wraps the HttpClient, is it worth creating a generic version of this method too?